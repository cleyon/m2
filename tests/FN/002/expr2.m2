@newcmd double{x}
Double @x@ is @expr 2*x@
@endcmd
@for i 1 3
@double{@{i}}
@next i
