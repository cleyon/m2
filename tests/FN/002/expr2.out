Double 1 is 2
Double 2 is 4
Double 3 is 6
